lte/gateway/fabfile.py

```diff
@@ -544,6 +543,7 @@ def _start_gateway_containerized():
 
     with cd(AGW_ROOT + "/docker"):
         # The `docker-compose up` times are machine dependent, such that a retry is needed here for resilience.
+        run('./run.py --snowflake')
```

Comment:
How about this solution?

Reply:
Honestly, I do not really see, why we are replacing one line of bash with roughly hundred lines of Python code. I cannot see any advantage there.
Can we not just place this snowflake file somewhere inside the installation and make sure it is there, instead of relying on some cryptic system dependency?